csfieldguide/chapters/content/en/human-computer-interaction/sections/usability-heuristics.md

```diff
@@ -5,24 +5,24 @@
 # HCI posters for your classroom!
 
 A set of posters that support this section have been published by Jennifer Gottschalk.
-[These are available in PDF format here]().
+These are available in PDF format here:
 
 {button-link link="files/HCI-posters.pdf" file="yes" text="Download HCI Posters"}
```

Comment:
Not related to your changes but the fact that this has a scroll bar is strange. I think the container for the button is wider than the panel.

![image](https://user-images.githubusercontent.com/16066822/60060385-764f7580-9744-11e9-8895-08d5b1b8c304.png)